extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -71,4 +71,30 @@ amp-carousel .amp-carousel-button.amp-disabled {
   animation: none;
   opacity: 0;
   visibility: hidden;
-}
\ No newline at end of file
+}
+
+/* Begin SlideScroll */
+.-amp-slidescroll {
+  /* remove the gap between inline(-block) elements */
+  font: 0/0 a;
```

Comment:
I'm even ok if it's broken now with extra space. But we certainly shouldn't do the obviously wrong thing here. To strip space, we can consider:

1. Researching what's the right `vertical-align` for `inline-block` here.
2. Not using `inline-block` and instead positioning slides in absolute way or via flexbox. Both should be fine, since in reality we will only be displaying three slides at a time, right?